configure: error / No package 'liblz4' found

Hi, 
I have a compilation Error.

configure: error: Package requirements (protobuf >= 2.4.0 libcurl >= 7.35.0 zlib >= 1.2.8 liblz4 >= 1.7.3) were not met:
No package 'liblz4' found

==> I installed the lib with the dockerfile (liblz4-dev).
but the header lz4frame.h is not found to compîle valhalla  file "valhalla_pack_elevation.cc"

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM debian:8
+FROM debian:9
 
 VOLUME /data/valhalla
 RUN apt-get update && apt-get install -y --no-install-recommends git \
@@ -20,6 +20,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends git \
       libspatialite-dev \
       libsqlite3-dev \
       spatialite-bin \
+      liblz4-dev \
       wget \
       unzip \
       lua5.2 \
@@ -27,20 +28,20 @@ RUN apt-get update && apt-get install -y --no-install-recommends git \
       vim-common \
       jq \
       gucharmap \
-      libprotobuf9 \
-      libgeos-3.4.2 \
+      libprotobuf10 \
+      libgeos-3.5.1 \
       liblua5.2 \
       libsqlite3-0 \
       libboost-all-dev \
-      libboost-date-time1.55.0 \
-      libboost-filesystem1.55.0 \
-      libboost-program-options1.55.0 \
-      libboost-regex1.55.0 \
-      libboost-system1.55.0 \
-      libboost-thread1.55.0 \
-      libboost-iostreams1.55.0 \
+      libboost-date-time1.62.0 \
+      libboost-filesystem1.62.0 \
+      libboost-program-options1.62.0 \
+      libboost-regex1.62.0 \
+      libboost-system1.62.0 \
+      libboost-thread1.62.0 \
+      libboost-iostreams1.62.0 \
       cmake \
-      libzmq-dev \
+      libzmq3-dev \
       ca-certificates \
   && git clone --depth=1 --recursive https://github.com/valhalla/valhalla.git libvalhalla \
   && cd libvalhalla \

--- a/asgard/asgard.cpp
+++ b/asgard/asgard.cpp
@@ -149,9 +149,6 @@ void worker(Context& context){
         std::vector<std::string> targets;
         targets.reserve(pb_req.sn_routing_matrix().destinations_size());
 
-        std::vector<valhalla::baldr::PathLocation> path_location_sources;
-        std::vector<valhalla::baldr::PathLocation> path_location_targets;
-
         for(const auto& e: pb_req.sn_routing_matrix().origins()){
             sources.push_back(e.place());
         }
@@ -167,12 +164,13 @@ void worker(Context& context){
         auto path_locations = projector(begin(range), end(range), graph, costing);
         LOG_INFO("Projecting locations done.");
 
+        google::protobuf::RepeatedPtrField<valhalla::odin::Location> path_location_sources;
+        google::protobuf::RepeatedPtrField<valhalla::odin::Location> path_location_targets;
         for (const auto& e: sources) {
-            path_location_sources.push_back(path_locations.at(e));
+            valhalla::baldr::PathLocation::toPBF(path_locations.at(e), path_location_sources.Add(), graph);
         }
-
         for (const auto& e: targets) {
-            path_location_targets.push_back(path_locations.at(e));
+            valhalla::baldr::PathLocation::toPBF(path_locations.at(e), path_location_targets.Add(), graph);
         }
 
         LOG_INFO("Computing matrix...");

